src/js/view/controls/controlbar.js

```diff
@@ -270,12 +278,15 @@ export default class Controlbar {
         _model.change('duration', this.onDuration, this);
         _model.change('position', this.onElapsed, this);
         _model.change('fullscreen', (model, val) => {
-            const fullscreenElement = this.elements.fullscreen.element();
-            toggleClass(fullscreenElement, 'jw-off', val);
-
-            const fullscreenText = model.get('fullscreen') ? localization.exitFullscreen : localization.fullscreen;
-            fullscreenTip.setText(fullscreenText);
-            setAttribute(fullscreenElement, 'aria-label', fullscreenText);
+            const fullscreenElements = [this.elements.fullscreen.element(), this.elements.adsFullscreen.element()];
+
+            for (let i=0; i < fullscreenElements.length; i++) {
```

Comment:
Lint will fail - need space before/after `=`

Reply:
It actually passes lint:styles (maybe we should add a rule) but I'll change it for convention